### PR TITLE
doc: translate an explanation of Groonga::Table into English

### PR DESCRIPTION
**Pull request by komainu8**
A part of a fix for an issue #52.

### CODE REVIEW

**Comment by komainu8** [comment]
@kenhys Could you review this PR, please?

File: ext/groonga/rb-grn-table.c [hunk 1]
```diff
@@ -184,7 +184,8 @@ rb_grn_table_inspect_content (VALUE self, VALUE inspected)
 }
 
 /*
- * テーブルの中身を人に見やすい文字列で返す。
+ * This function return contents of a table as a string
+ * easy to understand for human.
```
Comment (kenhys) [inline ext/groonga/rb-grn-table.c]: How about

`This function return contents of a table as a string.
It's easy to understand for human.`
or
`This function return contents of a table as a pretty-printed string.`

**Comment by komainu8** [comment]
@kenhys Thank you for your comment.
I've fixed it.

File: ext/groonga/rb-grn-table.c [hunk 2]
```diff
@@ -184,7 +184,8 @@ rb_grn_table_inspect_content (VALUE self, VALUE inspected)
 }
 
 /*
- * テーブルの中身を人に見やすい文字列で返す。
+ * This function return contents of a table as a
+ * pretty-printed string.
```
Comment (kou) [inline ext/groonga/rb-grn-table.c]: In Ruby, "pretty-print" is used for https://docs.ruby-lang.org/en/2.6.0/PP.html
It's different with "inspect": https://docs.ruby-lang.org/en/2.6.0/Object.html#method-i-inspect

Comment (komainu8) [reply ext/groonga/rb-grn-table.c]: Thank you for your comment.
I've fixed it in c09caa53e7c34c768558fd3249ca355da19a3c1c.

**Pull request merged**